docs/src/1_Mechanics/02_GMGuide_Brief.md

```diff
@@ -0,0 +1,50 @@
+# GM’s Quickstart Guide to Deck of Adventures
+
+## Introduction
```

Comment:
Having only one H2 makes this look unfinished. Either (a) remove this, promote subsequent headings, and add this doc to `mkdocs.yml`, or (b) shelve this doc until complete.

Reply:
Removed from current PR.